app/src/app/rc.js

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+import {app} from 'electron'
+import {existsSync} from 'fs'
+import loadJson from 'load-json-file'
+import {isWindows, isOSX} from './utils'
+
+// Directory where of .app or .exe file lives.
+let dir
+
+if (isOSX()) dir = path.normalize(app.getAppPath() + '/../../../..')
```

Comment:
from https://github.com/electron/electron/blob/master/docs/api/app.md#appgetpathname it seems that you could do `app.getAppPath('exe')`